app/views/distributions/show.html.erb

```diff
@@ -58,6 +58,8 @@
         <% end %>
         <%= f.submit "Confirm Distribution", class: "btn btn-lg btn-primary" %>
       <% end %>
+    <% else %>
+      <%= link_to "Make a correction", edit_distribution_path(@distribution), class: "btn btn-primary" %>
```

Comment:
Can you also dd a link to print a pdf of the distribution?

Reply:
Definitely would be good to have, but I think we should address that in a separate PR. Logging an issue.

Reply:
#477

Reply:
👍